test/boostTest.cpp

```diff
@@ -156,14 +155,27 @@ test_suite* init_unit_test_suite( int /*argc*/, char* /*argv*/[] )
 
 	initializeOptions();
 
-	bool disableSemantics = !solidity::test::EVMHost::getVM(solidity::test::CommonOptions::get().evmonePath.string());
-	if (disableSemantics)
+	bool disableSemantics = true;
+	try
 	{
-		cout << "Unable to find " << solidity::test::evmoneFilename << ". Please provide the path using -- --evmonepath <path>." << endl;
-		cout << "You can download it at" << endl;
-		cout << solidity::test::evmoneDownloadLink << endl;
+		disableSemantics = !solidity::test::EVMHost::checkVmPaths(solidity::test::CommonOptions::get().vmPaths);
+	}
+	catch (std::exception const& _exception)
+	{
+		cerr << "Error: " << _exception.what() << endl;
+		exit(1);
+	}
+	if (disableSemantics)
 		cout << endl << "--- SKIPPING ALL SEMANTICS TESTS ---" << endl << endl;
+
+	cout << "Loaded EVMC VM's:" << endl;
```

Comment:
Does this get annoying over time?

Reply:
I don't think so. I think it is good to see what evm's where actually loaded.

Reply:
If you request ewasm and it cannot be loaded, the test should exit with an error - or do you really want to know the exact filename where it loaded the shared library from?

Reply:
It was somehow useful while implementing.. but you're right, there is not much real value having this in general